ee/desktop/runner/runner.go

```diff
@@ -800,3 +819,42 @@ func removeFilesWithPrefix(folderPath, prefix string) error {
 		return os.Remove(path)
 	})
 }
+
+// IsAppindicatorEnabled iterates over known appindicator gnome-extensions for each user
+// and returns true the first time it finds one enabled, false if none found enabled
+func IsAppindicatorEnabled(ctx context.Context) bool {
+	uids, err := consoleuser.CurrentUids(ctx)
+	if err != nil {
+		return false
+	}
+
+	var extensions = []string{
+		"ubuntu-appindicators@ubuntu.com",
+		// appindicator that ships with NixOS
+		"appindicatorsupport@rgcjonas.gmail.com",
+	}
+
+	for _, extension := range extensions {
+		for _, uid := range uids {
+			ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+			defer cancel()
+
+			cmd := exec.CommandContext(ctx, "/usr/bin/gnome-extensions", "show", extension)
```

Comment:
Does `gnome-extensions list --enabled` let us do this with a single exec?

Reply:
yes! single exec per user

Reply:
updated